Hangman.c

```diff
@@ -0,0 +1,77 @@
+#include <stdio.h>
```

Comment:
Your code compiled and didn't have any warnings, which was great! You didn't follow directions in terms of the display, but nicely done, overall.